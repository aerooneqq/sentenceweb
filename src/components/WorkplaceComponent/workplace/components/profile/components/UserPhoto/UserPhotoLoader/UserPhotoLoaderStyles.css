.userPhotoLoaderOutterCont { 
    width: 100%;
    height: 100%;

    display: flex;
    flex-direction: column;

    justify-content: center;
    align-items: center;
}

.userPhotoLoaderTopBar { 
    width: 90%;
    height: 40px;

    margin-top: 15px;

    background: #d0d0d0;

    border-style: none;
    border-radius: 5px;
}

.userPhotoLoaderPhotoCont { 
    margin-top: 20px;

    width: 200px; 
    height: 200px;

    border-radius: 150px;

    background: #d0d0d0;
}