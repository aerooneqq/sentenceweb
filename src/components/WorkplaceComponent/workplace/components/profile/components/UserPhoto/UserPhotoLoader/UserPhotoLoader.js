import React, {Component} from "react";

//Styles
import "./UserPhotoLoaderStyles.css";


export default class UserPhotoLoader extends Component { 

    constructor(props) { 
        super(props);
    }

    render() { 
        return ( 
            <div className = "userPhotoLoaderOutterCont">

                <div className = "userPhotoLoaderTopBar"/>
                <div className = "userPhotoLoaderPhotoCont" />

            </div>
        )
    }
}