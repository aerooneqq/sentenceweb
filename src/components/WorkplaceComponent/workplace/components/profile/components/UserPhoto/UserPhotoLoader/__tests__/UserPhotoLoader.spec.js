import React from 'react'
import { shallow } from 'enzyme'
import {shallowToJson} from "enzyme-to-json"
import UserPhotoLoader from "../UserPhotoLoader";

describe("UserPhotoLoader", () => { 
    it("should render corectly", () => { 
        const userPhotoLoader = shallow(<UserPhotoLoader />);

        expect(shallowToJson(userPhotoLoader)).toMatchSnapshot();
    });

    it("should have userPhotoLoaderTopBar and userPhotoLoaderPhotoCont", () => { 
        const userPhotoLoader = shallow(<UserPhotoLoader />);

        expect(userPhotoLoader.find("div.userPhotoLoaderTopBar").exists()).toBe(true);
        expect(userPhotoLoader.find("div.userPhotoLoaderPhotoCont").exists()).toBe(true);
    });
});