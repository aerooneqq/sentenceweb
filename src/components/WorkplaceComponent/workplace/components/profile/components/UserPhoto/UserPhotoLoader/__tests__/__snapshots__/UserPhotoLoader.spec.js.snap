// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPhotoLoader should render corectly 1`] = `
<div
  className="userPhotoLoaderOutterCont"
>
  <div
    className="userPhotoLoaderTopBar"
  />
  <div
    className="userPhotoLoaderPhotoCont"
  />
</div>
`;
